templates/taxi/driver_list.html

```diff
@@ -8,6 +8,12 @@ <h1>
       </a>
     </h1>
 
+    <form action="" method="get" class="form-inline" novalidate>
+      {{ search_form|crispy }}
+      <input type="submit" value="Search" class="btn btn-secondary">
+    </form>
```

Comment:
Don't duplicate code, please. You can move it to new include html template and reuse it.